site/gatsby-node.js

```diff
@@ -33,10 +111,7 @@ exports.createPages = async ({ graphql, actions }) => {
       organizations: allAirtable(filter: { table: { eq: "Organizations" } }) {
         nodes {
           id
-          data {
-            Name
-            Role
-          }
+          ...OrganizationCard
```

Comment:
Mirroring capital organizations should be as easy as updating this query to use the `CapitalOrganizationCard` query, which merely adds a few properties to `OrganizationCard`.